pkg/orchestrator/stats.go

```diff
@@ -16,17 +17,17 @@ var (
 
 // CheckStats holds statistics for the DCA status command regarding the last run check. Information is saved in the KubernetesResourceCache.
 type CheckStats struct {
-	// CacheHits contains the number of cache hits for a NodeType per run.
+	// CacheHits contains the number of cache hits for a agentModel.K8SResources per run.
 	CacheHits int
 
-	// CacheMiss contains the number of cache miss/send Data for a NodeType per run.
+	// CacheMiss contains the number of cache miss/send Data for a agentModel.K8SResources per run.
 	CacheMiss int
 
-	NodeType
+	agentModel.K8SResources
```

Comment:
Would it make sense to keep the field name?
```suggestion
	NodeType agentModel.K8SResources
```
K8SResources is plural, which doesn't work very well for a field that contains one value only. (or `K8SResource`)

Reply:
Thanks for the feedback, I updated it to `agentModel.K8SResource`